perception/lidar_centerpoint/include/lidar_centerpoint/preprocess/pointcloud_densification.hpp

```diff
@@ -15,6 +15,8 @@
 #ifndef LIDAR_CENTERPOINT__PREPROCESS__POINTCLOUD_DENSIFICATION_HPP_
 #define LIDAR_CENTERPOINT__PREPROCESS__POINTCLOUD_DENSIFICATION_HPP_
 
+#include <lidar_centerpoint/cuda_utils.hpp>
```

Comment:
nit: [google style](https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes) for headers, especially using `""` instead of `<>` in project's headers. This comment refers to all files of this package.

Reply:
Thanks. Addressed in [f3187e9](https://github.com/autowarefoundation/autoware.universe/pull/6989/commits/f3187e9ce8364e86d4539b3ef1fa65223f6a35b9)

Note1: includes is a rabbit hole since even base ros packages are in a chaotic state: https://github.com/ros-perception/image_pipeline/blob/rolling/image_proc/src/rectify.cpp
Note2: The order specified by the google style guidelines can not be used since our pre-commit orders them again

Reply:
Sorry for not precise description: I meant `""` only for `lidar_centerpoint` project as it was shown in this [example](https://autowarefoundation.github.io/autoware-documentation/main/contributing/coding-guidelines/languages/cpp/#example).  Anyway, I'm looking into universe project and this pattern is not followed always, therefore fell free to resolve this conversation.